deploy-prod.sh

```diff
@@ -3,14 +3,15 @@
 # Copy or symlink it to some deployment directory.
 
 OPENFISCA_PORT=12000
+TARGET=${1:-origin/master}
 
 set -ex
 
 cd /var/www/dds
 
 # Update mes-aides
 git fetch origin
-git checkout --force --detach origin/master
+git checkout --force --detach $TARGET
```

Comment:
Wouldn't it better to have :
```sh
TARGET=${1:-master}
...
git checkout --force --detach origin/$TARGET
```
`./deploy.sh aah` seems more natural than ``./deploy.sh origin/aah``